offline/upgrading.md

```diff
@@ -1,8 +1,6 @@
 # How to upgrade wire (services only)
 
-We have a pipeline in  `wire-server-deploy` producing container images, static
-binaries, ansible playbooks, debian package sources and everything required to
-install Wire.
+We have a pipeline in  `wire-server-deploy` producing container images, static binaries, ansible playbooks, debian package sources and everything required to install Wire.
```

Comment:
Here is a superfluous space in the line. I'm not sure if it will be rendered or be removed by the markdown renderer of choice.